pkg/log/log.go

```diff
@@ -104,10 +103,11 @@ var (
 
 // InitLogger initializes DM's and also the TiDB library's loggers.
 func InitLogger(cfg *Config) error {
-	err := logutil.InitLogger(&logutil.LogConfig{Config: pclog.Config{Level: cfg.Level}})
-	if err != nil {
-		return terror.ErrInitLoggerFail.Delegate(err)
-	}
+	// currently only schema tracker use global logger(std logger)
```

Comment:
fix comment

Reply:
set and replace slow query logger if log level is debug

Reply:
we don't need to init schema-tracker logger because dm-worker and syncer aleady init logger in their main func

see https://github.com/pingcap/dm/blob/1334afefbb4ad9e424c34c7c4f251175b15368c1/cmd/dm-worker/main.go#L61

https://github.com/pingcap/dm/blob/1334afefbb4ad9e424c34c7c4f251175b15368c1/cmd/dm-syncer/main.go#L69